services/web/server/src/async_sio.py

```diff
@@ -121,27 +121,21 @@ def stop_dynamic_service(sid, data):
 
 @SIO.on('listObjects')
 async def list_S3_objects(sid, data):
-    _LOGGER.debug("client %s requests S3 objects in %s", sid, data)
+    _LOGGER.debug("client %s requests objects in storage. Extra argument %s", sid, data)
     _config = s3_config()
 
     s3_client = S3Client(endpoint=_config.endpoint,
         access_key=_config.access_key, secret_key=_config.secret_key)
 
     objects = s3_client.list_objects_v2(_config.bucket_name)
+    dataOut = []
```

Comment:
python uses snake case ... not sure why the linter is not catching thisone ... I wonder if it is again deactivated!:(